validation/validate_dc2_run2.2i_object_table.ipynb

```diff
@@ -0,0 +1,1367 @@
+    "# plt.text(18, 2, \"GALAXIES\", fontdict={'fontsize': 24}, color='orange')\n",
+    "plt.colorbar(label='objects / bin');"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.hist([df.loc[galaxy_idx, 'mag_g'] - df.loc[galaxy_idx, 'mag_r'], df.loc[star_idx, 'mag_g'] - df.loc[star_idx, 'mag_r']],\n",
+    "        label=['galaxies', 'stars'], histtype='step',\n",
+    "        bins=np.linspace(-5, +5, 501));\n",
+    "plt.xlabel('g-r')\n",
+    "plt.ylabel('objects / bin')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.hexbin(df.loc[star_idx, 'mag_g'] - df.loc[star_idx, 'mag_r'], df.loc[star_idx, 'mag_i'] - df.loc[star_idx, 'mag_i_cModel'],\n",
+    "           extent=(-2, +3, -0.5, +5),\n",
+    "           bins='log');\n",
+    "plt.xlabel('g-r')\n",
+    "plt.ylabel('mag_i[_psf] - mag_i_CModel');\n",
+    "# plt.text(14.5, 0.3, \"STARS\", fontdict={'fontsize': 24}, color='orange')\n",
+    "# plt.text(18, 2, \"GALAXIES\", fontdict={'fontsize': 24}, color='orange')\n",
+    "plt.colorbar(label='objects / bin');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Shape Parameters\n",
+    "\n",
+    "Ixx, Iyy, Ixy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_shape(df, filt, good_idx, star_idx, galaxy_idx, ax=None, legend=True):\n",
+    "    if not ax:\n",
+    "        ax = fig.gca()\n",
+    "\n",
+    "    names = ['good', 'star', 'galaxy']\n",
+    "    colors = ['blue', 'orange', 'green']\n",
+    "    hist_kwargs = {'color': colors, 'log': True,\n",
+    "             'bins': np.logspace(-1, 1.5, 100),\n",
+    "             'range': (0, 50),\n",
+    "             'histtype': 'step'}\n",
+    "    for prefix, ls in (('Ixx', '-'), ('Iyy', '--'), ('Ixy', ':')):\n",
+    "        field = f'{prefix}_{filt}'\n",
+    "        labels = [f'{prefix} {name}' for name in names]\n",
+    "        ax.hist([df.loc[good_idx, field], df.loc[star_idx, field], df.loc[galaxy_idx, field]],\n",
+    "                label=labels,\n",
+    "                linestyle=ls,\n",
+    "                **hist_kwargs)\n",
+    "\n",
+    "    ax.set_ylim(100, ax.get_ylim()[1])\n",
+    "    \n",
+    "    ax.set_xlabel(f'{filt}-band Moments: Ixx, Iyy, Ixy [pixels^2]')\n",
+    "    ax.set_ylabel('objects / bin')\n",
+    "    if legend:\n",
+    "        ax.legend()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axes = plt.subplots(2, 3, figsize=(12, 6))\n",
+    "legend = True\n",
+    "for ax, filt in zip(axes.flat, filters):\n",
+    "    plot_shape(df, filt, good_idx, star_idx, galaxy_idx, ax=ax, legend=legend)\n",
+    "    legend = False"
```

Comment:
After the for loop, add `plt.tight_layout()` so that the plot looks nicer (less overlapping text)